### PR TITLE
Resources branch top section code changes

### PR DESCRIPTION
**Pull request by aggarwalkartik555**

### CODE REVIEW

File: src/components/resources/data.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "imageUrl": images.resourceImg3,
+        'title': 'Classroom Culture',
+        'list': ['Learning App', 'Visual Study', 'Calender Apps', 'Audiobooks', 
+                'Podcasts', 'Tutoring Websites', 'Math and Science Simulations',
+                'E-book Platforms', 'Visual Study Groups', 'Online Libraries',
+            ]
+    }
+]
+
+export default data;
```
Comment (vigilantee) [inline src/components/resources/data.js:30]: add a new line here..

File: src/components/resources/resources.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            .resources-content{
+                flex-direction: column;
+                row-gap: 16px;
+                .resource-card {
+                    width: 352px;
+                }
+            }
+        }
+    }
+}
```
Comment (vigilantee) [inline src/components/resources/resources.css:102]: add a new line.

**Review by vigilantee — CHANGES_REQUESTED** [review]

File: src/components/resources/data.js (hunk 1 above)
Comment (aggarwalkartik555) [reply src/components/resources/data.js]: done

File: src/components/resources/resources.css (hunk 2 above)
Comment (aggarwalkartik555) [reply src/components/resources/resources.css]: done

**Review by vigilantee — APPROVED** [review]

**Pull request merged**